src/containers/Header/Header.tsx

```diff
@@ -2,37 +2,62 @@ import React from 'react'
 import "./header.scss"
 
 const Header = () => {
+  const menuItemLeft = [
+    {
+      id: 1,
+      title: "All Jobs",
+    },
+    {
+      id: 2,
+      title: "IT Companies",
+    },
+    {
+      id: 3,
+      title: "Blog",
+    }
+  ]
+  const renderMenuItemLeft = menuItemLeft.map((menuItem)=> {
+      return(<>
+        <li className="nav-item ms-xl-4" key={menuItem.id}>
+                  <a className="nav-link fs-15 text-shadow-white" aria-current="page" href="/#">{menuItem.title}</a>
+                </li>
+      </>)
+    })
+  const menuItemRight = [
+    {
+      id: 1,
+      title: "Sign in",
+    },
+    {
+      id: 2,
+      title: "IT Companies",
+    },
+    {
+      id: 3,
+      title: "Employers",
+    }
+  ]
+  const renderMenuItemRight = menuItemRight.map((menuItem)=> {
```

Comment:
`renderMenuItemLeft` and `renderMenuItemRight` are same logic. Please extract to one.